plugins/base.py

```diff
@@ -30,11 +30,13 @@ def on_load_end(self):
         function_origin = type(self).on_load_end.__qualname__.split(".")[0]
         print_checker(function_origin, f"{self.printprefix} on_load_end")
 
-    def on_fit_begin(self, store: Store, last_output: Any) -> Tuple[Store, Any]:
+    def on_fit_begin(
+        self, store: Store, last_output: Any
+    ) -> Tuple[Store, Any, List[Callable]]:
         function_origin = type(self).on_fit_begin.__qualname__.split(".")[0]
         print_checker(function_origin, f"{self.printprefix} on_fit_begin")
 
-        return store, last_output
+        return store, last_output, []
```

Comment:
now this is really hacky! so can a plugin override a previous plugin's List[Callable]?